### PR TITLE
Fix schema description errors

### PR DESCRIPTION
**Pull request by egierach**
For #256 

Also adds require for IdentityAssertion and NewIdentityAssertion in generator code, which we found missing when compiling everything.

### CODE REVIEW

File: src/ctim/generate.clj [hunk 1]
```diff
@@ -11,6 +11,7 @@
     [campaign :refer [Campaign NewCampaign]]
     [coa :refer [COA NewCOA]]
     [feedback :refer [Feedback NewFeedback]]
+    [identity-assertion :refer [IdentityAssertion NewIdentityAssertion]]
```
Comment (msprunck) [inline src/ctim/generate.clj]: Could you also add `:identity-assertion` and `:new-identity-assertion` in the `generators`?

The `IdentityAssertion` is also missing in the `ctim.document` ns

File: src/ctim/schemas/verdict.cljc [hunk 2]
```diff
@@ -13,13 +13,13 @@
...
-   "disposition has priority over all others, then Malicious disposition, and so "
-   "on down to Unknown.\n\n The ID of a verdict is a a str of the form "
-   "\"observable.type:observable.value\" for example, \"ip:1.1.1.1\"")
+  {:description (str
+                 "A Verdict is chosen from all of the Judgements on that Observable which have "
+                 "not yet expired.  The highest priority Judgement becomes the active verdict.  "
+                 "If there is more than one Judgement with that priority, then Clean "
+                 "disposition has priority over all others, then Malicious disposition, and so "
+                 "on down to Unknown.\n\n The ID of a verdict is a a str of the form "
+                 "\"observable.type:observable.value\" for example, \"ip:1.1.1.1\"")}
```
Comment (msprunck) [inline src/ctim/schemas/verdict.cljc]: This change is not needed. See my comment in the related issue.

File: src/ctim/schemas/judgement.cljc [hunk 3]
```diff
@@ -34,7 +34,7 @@
   intelligence data in CTIA.")
 
 (def-entity-type Judgement
-  judgement-desc
+  {:description judgement-desc}
```
Comment (msprunck) [inline src/ctim/schemas/judgement.cljc]: This change is not needed. See my comment in the related issue.

**Comment by noprompt** [comment]
This problem has been fixed in Flanders `v0.1.19`.

**Pull request closed without merging**